src/tables.rs

```diff
@@ -20,12 +20,20 @@
 // |1111|bad     |bad     |bad     |bad          |
 // |----|--------|--------|--------|-------------|
 pub static BITRATE_INDEX: [[u16; 15]; 5] = [
-    [0, 32, 64, 96, 128, 160, 192, 224, 256, 288, 320, 352, 384, 416, 448], // Version 1, Layer 1
-    [0, 32, 48, 56,  64,  80,  96, 112, 128, 160, 192, 224, 256, 320, 384], // Version 1, Layer 2
-    [0, 32, 40, 48,  56,  64,  80,  96, 112, 128, 160, 192, 224, 256, 320], // Version 1, Layer 3
-    [0, 32, 48, 56,  64,  80,  96, 112, 128, 144, 160, 176, 192, 224, 256], // Version 2, Layer 2
-    [0,  8, 16, 24,  32,  40,  48,  56,  64,  80,  96, 112, 128, 144, 160], // Version 2, Layer 2
-];                                                                          // & Layer 3
+    [
```

Comment:
Could you rustfmt ignore this table and revert it to the old version? The older version is better IMO.